packages/sdk/wallets/wallet-local/src/local-signer.ts

```diff
@@ -26,9 +24,7 @@ export class LocalSigner implements Signer {
     addToV: number,
     encodedTx: RLPEncodedTx
   ): Promise<{ v: number; r: Buffer; s: Buffer }> {
-    const hash = getHashFromEncoded(encodedTx.rlpEncode)
-    const signature = Account.makeSigner(addToV)(hash, this.privateKey)
-    return decodeSig(signature)
+    return signTransaction(getHashFromEncoded(encodedTx.rlpEncode), this.privateKey, addToV)
```

Comment:
so initially this method was signing and decoding the signature to get the signed transactions and now it's just... signing it? makes sense, but do you know the reason why it was doing it... around?

Reply:
oh it's still doing the same thing, signTransaction returns a `{v,r,s}` object just like before